### PR TITLE
NVSv4 - skip on linux since current it doesn't support on linux

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: lisa/features/gpu.py [hunk 1]
```diff
@@ -137,6 +137,9 @@ def install_compute_sdk(self, version: str = "") -> None:
                     except Exception as e:
                         raise LisaException(f"Failed to install CUDA Driver {str(e)}")
                     self.gpu_vendor.add("nvidia")
+            elif driver == ComputeSDK.AMD:
+                # refer https://learn.microsoft.com/en-us/azure/virtual-machines/nvv4-series  # noqa: E501
+                raise SkippedException("AMD GPU is not supported in Linux yet.")
```
Comment (squirrelsc) [inline lisa/features/gpu.py]: I cannot find any information about AMD GPU doesn't support Linux in this page. If so, please clarify, it's not supported on Azure, or general not support Linux.

Comment (LiliDeng) [reply lisa/features/gpu.py]: should be this link https://learn.microsoft.com/en-us/azure/virtual-machines/sizes-gpu

image

Comment (LiliDeng) [reply lisa/features/gpu.py]: this page https://learn.microsoft.com/en-us/azure/virtual-machines/nvv4-series also refers to Windows only
image

Comment (squirrelsc) [reply lisa/features/gpu.py]: Good to know. I think it means the NVv4 series supports windows only. it doesn't mean the AMD GPU supports Windows only. Can we exclude the VM sizes in GPU feature, and add comment to explain there?

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**